snmp/datadog_checks/snmp/data/default_profiles/alcatel-lucent-ent.yaml

```diff
@@ -0,0 +1,307 @@
+extends:
+  - alcatel-lucent.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.6486.801.1.1.2.1.*
+metrics:
+  - MIB: ALCATEL-ENT1-HEALTH-MIB
+    table:
+      name: healthModuleTable
+      OID: 1.3.6.1.4.1.6486.801.1.2.1.16.1.1.1.1
+    symbols:
+      - name: cpu.usage
+        OID: 1.3.6.1.4.1.6486.801.1.2.1.16.1.1.1.1.1.11
+    metric_tags:
+      - column:
+          name: healthModuleChassisId
+          OID: 1.3.6.1.4.1.6486.801.1.2.1.16.1.1.1.1.1.14
+        tag: health_module_chassis_id
+      - column:
+          name: healthModuleChassisId
+          OID: 1.3.6.1.4.1.6486.801.1.2.1.16.1.1.1.1.1.14
+        tag: cpu
```

Comment:
```suggestion
      - index: 1  # healthModuleSlot
        tag: cpu
```

Reply:
I think it's more reliable to tag by `index: 1  # healthModuleSlot` that is the index of the table.

Reply:
and it's more consistent with the other cases of `cpu` and `mem` tags.

Reply:
Makes sense, fixed!